### PR TITLE
Distribute files

### PR DESCRIPTION
**Pull request by KostinP**

### CODE REVIEW

**Comment by lalagvana** [comment]
Wrong namespaces.

**Review by lalagvana — APPROVED** [review]

File: test/CheckRightsService.Business.UnitTests/CheckRightsService.Business.UnitTests.csproj [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <AssemblyName>LT.DigitalOffice.CheckRightsService.Business.UnitTests</AssemblyName>
+
+    <RootNamespace>LT.DigitalOffice.CheckRightsService.Business.UnitTests</RootNamespace>
```
Comment (djino97) [inline test/CheckRightsService.Business.UnitTests/CheckRightsService.Business.UnitTests.csproj:10]: Remove empty line and do also in others project

**Review by djino97 — CHANGES_REQUESTED** [review]
Not started 
image

File: src/CheckRightsService.Business/CheckRightsService.Business.csproj [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <AssemblyName>LT.DigitalOffice.CheckRightsService.Business</AssemblyName>
+    <RootNamespace>LT.DigitalOffice.CheckRightsService.Business</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="FluentValidation" Version="9.0.1" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.3" />
```
Comment (djino97) [inline src/CheckRightsService.Business/CheckRightsService.Business.csproj:11]: update version for kernel

Comment (KostinP) [reply src/CheckRightsService.Business/CheckRightsService.Business.csproj]: Fixed.

File: test/CheckRightsService.Business.UnitTests/CheckRightsService.Business.UnitTests.csproj (hunk 1 above)
Comment (KostinP) [reply test/CheckRightsService.Business.UnitTests/CheckRightsService.Business.UnitTests.csproj]: Fixed.

File: src/CheckRightsService.Business/AddRightsForUserCommand.cs [hunk 3]
```diff
@@ -26,9 +27,17 @@ public class AddRightsForUserCommand : IAddRightsForUserCommand
...
+            var validationResult = validator.Validate(request);
 
-            if (!accessValidator.IsAdmin().Result)
+            if (validationResult != null && !validationResult.IsValid)
+            {
+                var messages = validationResult.Errors.Select(x => x.ErrorMessage);
+                string message = messages.Aggregate((x, y) => x + "\n" + y);
+
+                throw new ValidationException(message);
+            }
```
[the remark this answers is not in the document]
Comment (KostinP) [reply src/CheckRightsService.Business/AddRightsForUserCommand.cs]: Fixed.

File: src/CheckRightsService.Business/CheckRightsService.Business.csproj [hunk 4]
```diff
@@ -1,7 +1,21 @@
...
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <AssemblyName>LT.DigitalOffice.CheckRightsService.Business</AssemblyName>
+    <RootNamespace>LT.DigitalOffice.CheckRightsService.Business</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="FluentValidation" Version="9.2.2" />
```
Comment (djino97) [inline src/CheckRightsService.Business/CheckRightsService.Business.csproj:10]: remove

**Review by djino97 — APPROVED** [review]

File: src/CheckRightsService.Business/CheckRightsService.Business.csproj (hunk 4 above)
Comment (KostinP) [reply src/CheckRightsService.Business/CheckRightsService.Business.csproj]: Fixed.

**Pull request merged**